packages/storybook/.storybook/main.js

```diff
@@ -24,11 +24,11 @@ module.exports = {
       include: path.resolve(__dirname, '../'),
     });
 
-    // This is so the docs page for each component can populate the args table.
-    // The propTypes are stripped out in the built version, so we need to use
-    // the source files
-    config.resolve.alias['@department-of-veterans-affairs/component-library$'] =
-      path.resolve(__dirname, '../../react-components/src');
+    // // This is so the docs page for each component can populate the args table.
+    // // The propTypes are stripped out in the built version, so we need to use
+    // // the source files
+    // config.resolve.alias['@department-of-veterans-affairs/component-library$'] =
+    //   path.resolve(__dirname, '../../react-components/src');
```

Comment:
I commented this out so that I could test in storybook more confidently by knowing that I'm importing things in a way that the would be imported in `vets-website`. It will be reverted before merge.